src/Middleware/ValidateRequestBody.php

```diff
@@ -2,28 +2,28 @@
 
 declare(strict_types=1);
 
-namespace Linio\Common\Laminas\Middleware;
+namespace Linio\Common\Mezzio\Middleware;
 
-use Linio\Common\Laminas\Exception\Http\MiddlewareOutOfOrderException;
-use Linio\Common\Laminas\Exception\Http\RouteNotFoundException;
-
-use function Linio\Common\Laminas\Support\getCurrentRouteFromMatchedRoute;
-
-use Linio\Common\Laminas\Validation\ValidationService;
-use Mezzio\Router\RouteCollector;
-use Mezzio\Router\RouteResult;
+use Linio\Common\Mezzio\Exception\Http\MiddlewareOutOfOrderException;
+use Linio\Common\Mezzio\Exception\Http\RouteNotFoundException;
+use function Linio\Common\Mezzio\Support\getCurrentRouteFromMatchedRoute;
+use Linio\Common\Mezzio\Validation\ValidationService;
 use Psr\Http\Message\ResponseInterface;
 use Psr\Http\Message\ServerRequestInterface;
+use Mezzio\Router\RouteResult;
 use Psr\Http\Server\MiddlewareInterface;
 use Psr\Http\Server\RequestHandlerInterface;
 
 class ValidateRequestBody implements MiddlewareInterface
 {
     private ValidationService $validationService;
 
-    private RouteCollector $routes;
+    /**
+     * An array of zend-expressive routes.
+     */
+    private array $routes;
 
-    public function __construct(ValidationService $validationService, RouteCollector $routes)
+    public function __construct(ValidationService $validationService, array $routes)
```

Comment:
**suggestion:** add return type

Reply:
In the case of our __construct method, it might not be necessary to add return type declarations as it doesn't seem to have one. In this case, I think we could skip adding the return type declarations and still have a clear understanding of what the method returns based on its name and the comments in the code.